src/modules/poweraccent/PowerAccentModuleInterface/dllmain.cpp

```diff
@@ -132,13 +221,48 @@ class PowerAccent : public PowertoyModuleIface
         }
     }
 
+    virtual bool on_hotkey(size_t /*hotkeyId*/) override
+    {
+        if (m_enabled)
+        {
+            disable();
+            PlaySound(TEXT("Media\\Windows Notify Messaging.wav"), NULL, SND_FILENAME | SND_ASYNC);
```

Comment:
I think it doesn't make sense to have the runner process play a sound.
Although I see that it can't likely be the QuickAccent process because it would need to play while exiting...
Can we turn this into a setting that's turned off by default? Always on top also has this as a setting, for example.